covid_moonshot_ml/datasets/moonshot.py

```diff
@@ -1,19 +1,29 @@
-BASE_URL = "https://app.collaborativedrug.com/api/v1/vaults/5549/"
+VAULT_URL = "https://app.collaborativedrug.com/api/v1/vaults/5549/"
 ## All molecules with SMILES (public)
 ALL_SMI_SEARCH = "searches/9469227-zd2doWwzJ63bZYaI_vkjXg"
+## Noncovalent molecules with experimental measurements (from John)
+NONCOVALENT_SMI_SEARCH = "searches/9737468-RPSZ3XnVP-ufU6nNTJjZ_Q"
 
+from io import StringIO
+import logging
+import pandas
 
-def download_url(url, header):
+
+def download_url(search_url, header, timeout=5000, retry_delay=5):
     """
     Make requests to the API using the passed information.
 
     Parameters
     ----------
-    url : string
+    search_url : string
         URL for the initial GET request
     header : dict
         Header information passed to GET request. Must contain an entry for
         'X-CDD-token' that gives the user's CDD API token
+    timeout : int, default=5000
```

Comment:
Not a blocker. If I'm reading this correctly, do we really want to try this ~1000 times by default?

Reply:
I think in general it'll take way less time than that, but I think the idea is to give the CDD server enough time to prep the download while also making sure to check often enough. there's no real basis to these numbers though so feel free to suggest alternatives if you have any thoughts on it

Reply:
This is fine then, I was just making sure the numbers made sense, since this would mean that we can wait up to 5000 (~1h 20mins) for things to get resolved, and was just wondering if that's a reasonable time to be waiting for this.

Reply:
Honestly, I'm not really sure. I thought I pulled this number from an example on the CDD API docs, but I just went to look for it and I couldn't find it so it's possible I made it up